### ISSUE TITLE
reTerminal LCD has orientation problem with auto-rotate function in RPi OS Bullseye (Kernel 5.15)

### ISSUE THREAD

**Issue by lakshanthad** [Seeed-Studio/seeed-linux-dtoverlays#43]
**Describe the bug**
After flashing the reTerminal with the latest Raspberry Pi OS Bullseye found [here](https://www.raspberrypi.com/software/operating-systems/), the LCD has orientation problem with auto-rotate function.

**Expected behavior**
It works well with RPi OS Buster

**Tests performed**

Test 1 with Bullseye and [these changes](https://wiki.seeedstudio.com/reTerminal-FAQ/#q11-the-screen-orientation-is-incorrect-after-installing-raspberry-pi-os-bullseye):

1. As soon as overlays are installed, the screen looks like this:

<div align=center><img width=300 src="https://user-images.githubusercontent.com/20147381/166867716-8abdc4b8-7203-4314-b057-1f35f6b5ed0a.jpg"/></div>

2. After following [these instructions](https://wiki.seeedstudio.com/reTerminal-FAQ/#q11-the-screen-orientation-is-incorrect-after-installing-raspberry-pi-os-bullseye) and reboot the device, the device orientation looks normal

<div align=center><img width=500 src="https://user-images.githubusercontent.com/20147381/166867916-9d4b197c-370a-4b30-85d2-232d88b350e6.jpg"/></div>

3. However, when I lift the device and place the device in landscape orientation, the orientation changes

<div align=center><img width=500 src="https://user-images.githubusercontent.com/20147381/166867989-99a5af8c-bd00-480e-8c03-06e6a6d79700.jpg"/></div>

4. Landscape (flipped) has same issue 

<div align=center><img width=500 src="https://user-images.githubusercontent.com/20147381/166868039-c9024f0c-481d-423a-b720-a0c9fe80a48e.jpg"/></div>

5. Portrait and portrait (flipped) have no issue 

<div align=center><img width=300 src="https://user-images.githubusercontent.com/20147381/166868085-3203d808-3562-4f53-84ab-480b9b2b1bd7.jpg"/></div>

<div align=center><img width=300 src="https://user-images.githubusercontent.com/20147381/166868091-814e94dc-54c1-4af7-9619-9e86d17d7b2a.jpg"/></div>

Test 2 with Bullseye without any changes:

1. As soon as overlays are installed, the screen looks like this:

<div align=center><img width=300 src="https://user-images.githubusercontent.com/20147381/166867716-8abdc4b8-7203-4314-b057-1f35f6b5ed0a.jpg"/></div>

2. However, even without following [this](https://wiki.seeedstudio.com/reTerminal-FAQ/#q11-the-screen-orientation-is-incorrect-after-installing-raspberry-pi-os-bullseye), the screen behaves the same as 2,3,4,5 in test 1 above.

In conclusion, no matter whether we follow [this](https://wiki.seeedstudio.com/reTerminal-FAQ/#q11-the-screen-orientation-is-incorrect-after-installing-raspberry-pi-os-bullseye) or not, the screen behavior for auto-rotate is the same.

Thank you.

Best Regards,
Lakshantha

**Comment by lakshanthad** [comment]
It is fixed here:
https://github.com/Seeed-Studio/seeed-linux-dtoverlays/commit/fc790132fe595166fd41cdb17cbdaf4104410e2e

Thank you very much for your support @bigbearishappy
